Makefile

```diff
@@ -384,14 +387,16 @@ switch-tanzu:
 #################################
 docker-multiarch-build: go-generate generate-config-file 
 	@ hack/docker.sh -l -m $(OPERATOR_IMAGE)
+	@ hack/docker.sh -l -m $(OPERATOR_DOCKERHUB_IMAGE)
 	docker buildx build . \
 		--progress=plain \
 		--build-arg GO_LDFLAGS='$(GO_LDFLAGS)' \
 		--build-arg GO_TAGS='$(GO_TAGS)' \
 		--build-arg VERSION='$(VERSION)' \
 		--platform linux/amd64,linux/arm64 \
-		--push \
-		-t $(OPERATOR_IMAGE)
+		-t $(OPERATOR_IMAGE) \
+		-t $(OPERATOR_DOCKERHUB_IMAGE) \
```

Comment:
Should we also update `build-operator-multiarch-image` to check if `OPERATOR_DOCKERHUB_IMAGE` has been published ? Right now it is only checking for `OPERATOR_IMAGE`:

```Makefile
build-operator-multiarch-image:
	@ docker buildx imagetools inspect $(OPERATOR_IMAGE) | grep -q 'linux/arm64' 2>&1 >/dev/null \
	&& echo "OK: image $(OPERATOR_IMAGE) already published" \
	|| $(MAKE) docker-multiarch-build
```

Reply:
yeah, I initially had this in a separate target to make this cleaner, but it was removed in a suggested change.  I'm adding it back here.  Testing in local jenkins to ensure I didn't bork it.